benches/nexmark/main.rs

```diff
@@ -285,5 +287,39 @@ fn main() -> Result<()> {
         ]
     }));
 
+    if nexmark_config.output_csv {
+        // The file is truncated if it already exists.
+        let mut csv_writer = csv::Writer::from_path(NEXMARK_RESULTS_FILE)?;
+        csv_writer.write_record(RESULT_COLUMNS)?;
+        for result in results.into_iter() {
+            // Converting units to seconds, bytes
+            let (before, after) = (result.before_stats, result.after_stats);
+            csv_writer.write_record(vec![
```

Comment:
Couldn't you use [`Writer::serialize`](https://docs.rs/csv/latest/csv/struct.Writer.html#method.serialize) for this?

Reply:
Will try and fix it in the next PR

Reply:
Fixed it in https://github.com/vmware/database-stream-processor/pull/222